android/KMAPro/build.sh

```diff
@@ -19,23 +14,51 @@ THIS_SCRIPT="$(readlink -f "${BASH_SOURCE[0]}")"
 . "$KEYMAN_ROOT/resources/build/build-help.inc.sh"
 . "$KEYMAN_ROOT/resources/build/build-download-resources.sh"
 
-echo Build KMAPro
-
-#
-# Prevents 'clear' on exit of mingw64 bash shell
-#
-SHLVL=0
-
-display_usage ( ) {
-    echo "build.sh [-no-daemon] [-debug] [-download-resources]"
-    echo
-    echo "Build Keyman for Android"
-    echo "  -no-daemon              Don't start the Gradle daemon. Use for CI"
-    echo "  -upload-sentry          Uploads debug symbols, etc, to Sentry"
-    echo "  -debug                  Compile only Debug variant"
-    echo "  -download-resources     Download sil_euro_latin.kmp and nrc.en.mtnt.model.kmp from downloads.keyman.com"
-    exit 1
-}
+# This script runs from its own folder
+cd "$THIS_SCRIPT_PATH"
+
+# ################################ Main script ################################
+
+# Definition of global compile constants
+
+CONFIG="release"
+BUILD_FLAGS="build -x lint -x test"                     # Gradle build w/o test
+TEST_FLAGS="-x assembleRelease lintRelease testRelease" # Gradle test w/o build
+DAEMON_FLAG=
+
+builder_describe "Builds Keyman for Android app." \
+  "@../KMEA" \
+  "clean" \
+  "configure" \
+  "build" \
+  "test             Runs lint and unit tests." \
+  "--ci             Don't start the Gradle daemon. For CI" \
+  "--upload-sentry  Upload to sentry"
+
+# parse before describe_outputs to check debug flags
+builder_parse "$@"
+
+if builder_is_debug_build; then
+  builder_heading "### Debug config ####"
+  CONFIG="debug"
+  BUILD_FLAGS="assembleDebug -x lint -x test"
+  TEST_FLAGS="-x assembleDebug lintDebug testDebug"
+fi
+
+builder_describe_outputs \
+  configure     kMAPro/libs/keyman-engine.aar \
+  build         kMAPro/build/outputs/apk/$CONFIG/keyman-${VERSION}.apk
```

Comment:
```suggestion
  configure     /android/KMAPro/kMAPro/libs/keyman-engine.aar \
  build         /android/KMAPro/kMAPro/build/outputs/apk/$CONFIG/keyman-${VERSION}.apk
```

Those path casing changes make me squirm! But I presume we will be renaming `/android/KMAPro` to `/android/app` at some point?